packages/blade/src/components/ActionList/ActionListBox.native.tsx

```diff
@@ -23,17 +23,17 @@ const _ActionListBox = React.forwardRef<SectionList, ActionListBoxProps>(
   ({ sectionData, actionListItemWrapperRole, isMultiSelectable, isInBottomSheet }, ref) => {
     const { footerHeight, setContentHeight } = useBottomSheetContext();
 
-    const renderActionListItem = React.useCallback(({ item }) => {
+    const renderActionListItem = React.useCallback(({ item }: any) => {
```

Comment:
wouldn't this make `renderActionListItem`'s type any as well? should it be React.ReactElement instead?